src/features/landing/components/AuthSection/AuthForm/AuthForm.tsx

```diff
@@ -152,7 +152,7 @@ function AuthForm(): React.ReactElement {
                       <UiLink href="https://github.com/VilnaCRM-Org" target="_blank">
                         Privacy Policy
                       </UiLink>
-                      та
+                      and the
                       <UiLink href="https://github.com/VilnaCRM-Org" target="_blank">
                         Use Policy
```

Comment:
and here localization

Reply:
There is localization here. You can check this on the Ukrainian version of the site. The i18next functionality is used here for text replacement. Here is the code from the localization file.

`"fullText": "I have read and accept the <1>Privacy Policy</1> and the <1>Use Policy</1> VilnaCRM Service"`

Its functionality from Trans component.